script/upload-crowdin-glossary.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv-safe').load()
+
+const globals = require('globals')
+const nodeCoreModules = require('builtins')('10.0.0')
```

Comment:
Maybe we could also collect builtins from other major versions like 8 and 6?

Reply:
According to the `builtins` source code https://github.com/juliangruber/builtins/blob/master/index.js, this setup will include builtins from 
 versions 6 and 8.